VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-telephone-id/copy-telephone-id.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ClipboardService } from 'ngx-clipboard';
+
+@Component({
+    selector: 'app-copy-telephone-id',
+    templateUrl: './copy-telephone-id.component.html',
+    styleUrls: ['./copy-telephone-id.component.scss']
+})
+export class CopyTelephoneIdComponent implements OnInit {
+    @Input() telephoneId: string;
+    tooltip: string;
+
+    constructor(private clipboardService: ClipboardService) {}
+
+    ngOnInit(): void {
+        this.tooltip = 'Copy joining by phone details to clipboard';
+    }
+
+    copyToClipboard() {
+        this.clipboardService.copyFromContent(this.telephoneId);
+        this.tooltip = 'Details copied to clipboard';
```

Comment:
If needed to copy again when tooltip opened would it still say 'Details copied to clipboard'?
I cant see anything resetting the state when the tooltip opens or closes?